Lamouresparus/Asa
# Asa 29 implement reading timer UI

### Tracked by [JIRA-29](https://loveotudor.atlassian.net/browse/ASA-29)
 
### This PR adds

The UI and functionality for a User to set up a timer to track the amount of time he spends reading a course.

### Considerations and implementation

We have added a User Interface to display a count-up timer to track the reading progress of a user, Once a user has set up the timer, the user would see a count Up timer showing the amount of time he has spent reading, the user can leave the app and  the timer would continue counting in the background, 
Once the User is not active in the app, he would receive a notification of the amount of time he has been reading, he can also click on the notification to return back to the Reading Timer UI

### How to Test

1. Launch the App
2. On the home screen, click on the "Profile Icon": should see a new screen displaying the user's profile.
3. On the Profile Screen, click any one of the course, a UI for the reading timer should appear
4. On the reading timer UI Screen, click "start the reading button", A count-up timer should start.
5. While not in the app, A Notification would appear, showing the amount of time you have spent reading
6. To stop tracking your reading time, click the stop button on the notification
7. To return back to the app, click on the Notification

### Test Added
N/A
